toolbox/Dockerfile

```diff
@@ -15,7 +15,6 @@
 FROM ubuntu:xenial
 MAINTAINER Jared Watts <jared.watts@quantum.com>
 
-
 RUN apt-get update && \
```

Comment:
Here's a curated list of changes in Kraken grepped by keyword "tool".  At a glance only a handful of Ceph admins using rook and tweaking their cluster with rook-tools might not find what they need with Jewel rools.

https://gist.github.com/bzub/030c9766b8124756a44d93adfb166c90